content/docs/handling-events.md

```diff
@@ -1,43 +1,46 @@
 ---
 id: handling-events
-title: Handling Events
+title: Omgaan met Events
 permalink: docs/handling-events.html
 prev: state-and-lifecycle.html
 next: conditional-rendering.html
 redirect_from:
   - "docs/events-ko-KR.html"
 ---
 
-Handling events with React elements is very similar to handling events on DOM elements. There are some syntactic differences:
+Omgaan met events met React-elementen lijkt sterk op omgaan met events met DOM-elementen.
+Er zijn enkele syntactische verschillen:
 
-* React events are named using camelCase, rather than lowercase.
-* With JSX you pass a function as the event handler, rather than a string.
+* React events worden in camelCase benoemd in plaats van kleine letters.
```

Comment:
```suggestion
* React events worden in camelCase geschreven in plaats van kleine letters.
```

Reply:
I would make it clear that it's about the name of React events:
```suggestion
* De naam van React events wordt in camelCase geschreven in plaats van kleine letters.
```
or do you think your version implies this enough?

Reply:
Yours is better. I lost the implicit meaning of "naming" of the original content. Nice one.